src/content/en/updates/posts/2016/01/urlsearchparams.markdown

```diff
@@ -0,0 +1,161 @@
+---
+layout: updates/post
+title: "Easy URL manipulation with URLSearchParams"
+description: "URLSearchParams trivial access and manipulation of the URL's query string"
+published_on: 2016-01-22
+updated_on: 2016-01-22
+authors:
+  - ericbidelman
+tags:
+  - url
+featured_image: /web/updates/images/2016/01/urlsearchparams/overview.png
+---
+
+<p class="intro">The <code>URLSearchParams</code> API provides a consistent
+interface to the bits and pieces of the URL and allows trivial manipulation
+of the query string (that stuff after "<code>?</code>").</p>
+
+Traditionally, developers use regexs and string splitting to pull out query
+parameters from the URL. If we're all honest with ourselves, that's no fun.
+It can be tedious and error prone to get right. One of my dark secrets is that I've reused the same
+`get|set|removeURLParameter` [helper methods](https://github.com/GoogleChrome/ioweb2015/blob/21d7a80aefd6a76474fcdb700ac9965cd4c7800f/app/scripts/helper/util.js#L142-L200) in several large Google.com app, including [Google Santa Tracker](https://santatracker.google.com/) and the [Google I/O 2015 web](https://events.google.com/io2015).
+
+It's time for a proper API that does this stuff for us!
+
+## URLSearchParams API
+
+[Try the demo](https://googlechrome.github.io/samples/urlsearchparams/index.html)
+
+Chrome 49 implements `URLSearchParams` from the [URL spec](https://url.spec.whatwg.org/#urlsearchparams), an API which is useful for fiddling around with URL
```

Comment:
Is it worth being explicit and say "fiddling around with the URL QueryString parameters"